node/src/tasks/bsp_upload_file.rs

```diff
@@ -250,50 +250,79 @@ where
             }
         };
 
-        // Query runtime for the chunks to prove for the file.
-        let chunks_to_prove = self
+        let own_bsp_id = self
```

Comment:
Shouldn't this be called `own_bsp_account`? Or `own_bsp_account_id`. Because `own_bsp_id` makes it sound as if it were the BSP ID.